wikimedia/wikipedia-ios
# Image Recommendations  - Add overflow menu

**Phabricator:** partially https://phabricator.wikimedia.org/T358902

### Notes
* Not including tutorial

### Test Steps
1. Go to image recs on Experimental
2. Click on the overflow menu
3. "Learn more" should open our FAQ in a  in-app webview 
4. Tutorial has no action for now, to be implemented 
5. "Report issue"should prompt the email client with a pre populated email body
